helm-charts/explorer/templates/explorer-db-service.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.db.name }}
+  labels:
+    name: {{ $.Values.db.name }}
+    app.kubernetes.io/name: {{ $.Values.db.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }}      
+spec:
+  type: {{ .Values.db.service.type }}
+  selector:
+    app: {{ .Values.db.name }}
+  ports:
+  - port: {{ .Values.db.service.port }}
+    targetPort: {{ .Values.db.service.port }}
+
+---
+{{- if .Values.db.ingress.enabled -}}
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Values.db.name }}
+  annotations: {{ $.Values.explorer.ingress.annotations }}
```

Comment:
Use the correct way of passing the annotation, this  doesn't work

Reply:
updated as per comments